### PR TITLE
Add recipe for Xerces-C++ which is needed by GDAL for the GML driver

### PR DESCRIPTION
**Pull request by gillins**
This recipe builds the Xerces-C++ parser. The build process is a bit messy on Windows but does work fine.

### CODE REVIEW

**Comment by groutr** [comment]
I think we need to add a build/run dependency on curl or else disable networking with `--disable-network`.  The recipe fails on my build machines because it tries to link against curl.  One other thing, we might want to use `--disable-static` since we try to link everything dynamically.

**Comment by gillins** [comment]
Agreed. We don't need the network support as far as I can tell.

File: xerces-c/bld.bat [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+) else (
+    set VCVERSION=VC9
+)
+
+msbuild projects\Win32\%VCVERSION%\xerces-all\xerces-all.sln /t:all /p:Configuration=Release;Platform=%PLATF%
+if errorlevel 1 exit 1
+
+REM implib and DLL
+mkdir %PREFIX%\lib
+copy Build\%OUTDIR%\%VCVERSION%\Release\xerces-c_3.lib %PREFIX%\lib\
```
Comment (groutr) [inline xerces-c/bld.bat]: `%PREFIX%` should be  `%LIBRARY_PREFIX%`

File: xerces-c/bld.bat [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+if errorlevel 1 exit 1
+copy Build\%OUTDIR%\%VCVERSION%\Release\xerces-c_3_1.dll %PREFIX%\bin\
```
Comment (groutr) [inline xerces-c/bld.bat]: `%PREFIX%` -> `%LIBRARY_PREFIX%`

File: xerces-c/bld.bat [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+if errorlevel 1 exit 1
+
+REM Headers. 
+echo .cpp > excludelist.txt
+mkdir %PREFIX%\include\xercesc
```
Comment (groutr) [inline xerces-c/bld.bat]: `%PREFIX%` -> `%LIBRARY_PREFIX%`

File: xerces-c/bld.bat [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+    set VCVERSION=VC10
+) else (
+    set VCVERSION=VC9
+)
+
+msbuild projects\Win32\%VCVERSION%\xerces-all\xerces-all.sln /t:all /p:Configuration=Release;Platform=%PLATF%
+if errorlevel 1 exit 1
+
+REM implib and DLL
+mkdir %PREFIX%\lib
```
Comment (groutr) [inline xerces-c/bld.bat]: `%PREFIX%` -> `%LIBRARY_PREFIX`

**Comment by groutr** [comment]
After fixing the prefixes on windows, it should be mergeable.

**Comment by gillins** [comment]
I have used %LIBRARY_BIN%, %LIBRARY_LIB% and %LIBRARY_INC% for Xerces-c instead. Is it ok to assume these dirs exist?
Also updated the GEOS recipe to use %LIBRARY_PREFIX%

**Comment by groutr** [comment]
Conda build takes care of creating those directories.
Everything looks alright to me.

**Pull request merged**